leftwm/release/PKGBUILD

```
@@ -0,0 +1,61 @@
+# Maintainer: Lex Childs <lexchilds@gmail.com>
+# Contributor: éclairevoyant
+
+pkgbase=leftwm
+pkgname=(leftwm leftwm-nonsystemd)
+pkgver=0.5.1
+pkgrel=2
```

```suggestion
pkgrel=1
```
We want to initialise the pakage bump with value `1` as increments only should happen on changes of the packaging script itself while the version stays unchanged.

---

Good callout! I think that this is corrected in the process of building/updating sha as well but not 100% certain of that fact.